jsapp/js/stores/useSession.tsx

```diff
@@ -0,0 +1,49 @@
+import sessionStore from './session';
+import {useEffect, useState} from 'react';
+import {reaction} from 'mobx';
+import type {AccountResponse} from '../dataInterface';
+
+/**
+ * Hook to use the session store in functional components.
+ * This hook provides a way to access teh current logged account, information
+ * regarding the anonymous state of the login and session methods.
+ *
+ * This hook uses mob-x reactions to track the current account and update the
+ * state accordingly.
+ * In the future we should update this hook to use react-query and drop the usage of mob-x
```

Comment:
I would change "mob-x" to "mobx" (the package name) or "MobX" (the name they use on their website), as the dash name is not being used :)

Reply:
changed!